model/event.go

```diff
@@ -75,9 +87,15 @@ type EventRef struct {
 	TriggerEventRef string `json:"triggerEventRef" validate:"required"`
 	// Reference to the unique name of a 'consumed' event definition
 	ResultEventRef string `json:"resultEventRef" validate:"required"`
+	// Maximum amount of time (ISO 8601 format) to wait for the result event. If not defined it will be set to the 'actionExecTimeout'
+	ResultEventTimeout string `json:"resultEventTimeout,omitempty" validate:"omitempty,min=1"`
```

Comment:
I think we can get rid of this `validate`. You're already enforcing it in the custom validator.